### DISCUSSION TITLE
Issue : Can't add credentials on Render and Hugginface

### DISCUSSION THREAD

**Discussion by Lachkar-Ahmed-Salim** [FlowiseAI/Flowise#1533] [Q&A] [answered]
Hi Everyone ,
First of all I want to give you a special thanks for this amazing tool , it’s like a dream come true for non-developers like me.
Here is the situation :
I installed everything according to the documentation  in Render first then on Huggingface , I can pretty much do anything with Flowise on both flowise and huggingface except for one thing , which is adding a credential , as you can see in the picture you can even get the suggestions for the  API keys to be added all what I get is this white empty window , also I also tried to add the credentials directly from the chatflows it’s the same problem. please help it’s for an emergency project.
Thank You 
![Screenshot 2024-01-16 003536](https://github.com/FlowiseAI/Flowise/assets/112639221/9ae15ff6-6764-4dcc-9c74-394c07152c38)

**Comment by Jai2899** [comment]
Facing the same issue. Not able to add any credentials.

**Reply by Lachkar-Ahmed-Salim** [reply, marked as answer]
Hi everyone , Flowise lanched a new patch that fixes this issue. https://github.com/FlowiseAI/Flowise/releases/tag/flowise%401.4.10 , I tried it and its working

**Comment by jmelati** [comment]
Same here. Can´t add credentials.

**Comment by Matija-S** [comment]
wow glad I am not the only one with this problem. Also can't add credenitals, just an empty search that does nothing

**Reply by Lachkar-Ahmed-Salim** [reply]
Hi everyone , Flowise lanched a new patch that fixes this issue. https://github.com/FlowiseAI/Flowise/releases/tag/flowise%401.4.10 , I tried it and its working

**Comment by Lachkar-Ahmed-Salim** [comment]
![Screenshot 2024-01-16 144029](https://github.com/FlowiseAI/Flowise/assets/112639221/a874d455-687e-43c4-bd93-a08c7bf83ff2)
I analysed my log this found out this error somehow its connected to upstash redis , and yet doesnt not exlain the credential problem do you guys got the same issue ?

**Reply by JuanMonederoLabs** [reply]
Same problem here. Using Render.

**Reply by Jai2899** [reply]
In package.json I changed @upstash/redis: "^1.22.1" to @upstash/redis:"1.22.1" and deployed it. Now I can search and add credentials.
![image](https://github.com/FlowiseAI/Flowise/assets/40239207/7cb0985b-8622-4b63-84c2-fe55c24fc0de)

**Reply by Lachkar-Ahmed-Salim** [reply]
@Jai2899 u made this change on packages/components/package.json directory ?

**Reply by SmartMart69** [reply]
Crazy - that worked for me...thank you

**Reply by Jai2899** [reply]
Yes @Hopeforzbest

**Reply by Lachkar-Ahmed-Salim** [reply]
Hi everyone , Flowise lanched a new patch that fixes this issue. https://github.com/FlowiseAI/Flowise/releases/tag/flowise%401.4.10 , I tried it and its working

**Comment by Lachkar-Ahmed-Salim** [comment]
Hi everyone , Flowise lanched a new patch that fixes this issue. https://github.com/FlowiseAI/Flowise/releases/tag/flowise%401.4.10 , I tried it and its working

**Comment by ahasibrm** [comment]
I'm having the same credential error in flowise 1.5.0, which presumably has the same fix as 1.4.10
